org/views.py

```diff
@@ -93,3 +93,41 @@ def AddVolunteer(request,org_id):
             return Response({"message":"You are added as a volunteer"},status.HTTP_201_CREATED)
     else:
         return Response({"detail":"Organization not present"},status.HTTP_400_BAD_REQUEST)
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
+    if org_count>0:
+        org = Org.objects.get(pk=org_id)
+        user = request.user
+        if Member.objects.filter(user=user,org=org).count()>0:
+            isadmin = Member.objects.get(user=user,org=org).group.perm_obj.permissions_to_integer()
+            #Checking if the isadmin is odd or even, if odd then the IS_ADMIN permission is enabled for the user
+            if isadmin%2 == 1:
+                if request.method == "PUT":
+                    serializer = EditOrgSerializer(org,data=request.data)
+                    if serializer.is_valid():
+                        serializer.save()
+                        return Response(responses.update_org_200,status.HTTP_200_OK)
+                    else:
+                        return Response(serializer.errors,status.HTTP_400_BAD_REQUEST)
+            else:
```

Comment:
No need of adding else here, if the upper response has been returned, it won't execute this code. 
@naveennvrgup What do you say, I am not clear about the standards here, but I think else can be removed as the previous condition may return or pass.

Reply:
>naveennvrgup ECellNitrr/managers What do you say, I am not clear about the standards here, but I think else can be removed as the previous condition may return or pass.

1. Yes, the else is redundant here @sahil9001 should fix it.  
1. @BadduCoder Feel free to point out or suggest any standard :v: wherever you seem fit. 
1. Sometimes I intentionally leave out a few minor issues which will be converted into `good first issue` in case if the future reviewers don't mention it out until the PR is merged. :sweat_smile: .

Reply:
@naveennvrgup Actually I have an exam tomorrow, so I wouldn't be able to do this today.

Reply:
@BadduCoder Why won't serializer.errors() will be returned? It's returning that in my tests